### PR TITLE
Use common base URL for KMT images

### PR DESCRIPTION
**Pull request by gjulianm**
### What does this PR do?

Fixes how images are defined in the `platforms.json` file and how they are downloaded. This fixes inconsistencies in how the images were downloaded (for example, sometimes the branch name would be ignored and replaced with master).

### Motivation

Consistency in how images are referred to and downloaded.

### CODE REVIEW

**Comment by gjulianm** [comment]
/trigger-ci --variable RUN_ALL_BUILDS=true --variable RUN_KITCHEN_TESTS=true --variable RUN_E2E_TESTS=on --variable RUN_UNIT_TESTS=on

File: test/new-e2e/system-probe/config/platforms.json [hunk 1]
```diff
@@ -1,36 +1,36 @@
 {
-    "url_base": "https://dd-agent-omnibus.s3.amazonaws.com/kernel-version-testing/rootfs",
+    "url_base": "https://dd-agent-omnibus.s3.amazonaws.com/kernel-version-testing/rootfs/master",
```
[the remark this answers is not in the document]
Comment (gjulianm) [reply test/new-e2e/system-probe/config/platforms.json:2]: Will change it then and allow setting per-distro branches

[the remark this answers is not in the document]
Comment (gjulianm) [reply test/new-e2e/system-probe/config/platforms.json]: Done!

**Review by usamasaqib — APPROVED** [review]

**Pull request merged**